client/src/services/statusService.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+
+class StatusService {
+  static getStatus() {
+    return axios({
+      method: 'get',
+      url: process.env.REACT_APP_STATUS_SERVICE,
+      xsrfCookieName: process.env.REACT_APP_AUTH_TOKEN,
+      xsrfHeaderName: process.env.REACT_APP_AUTH_HEADER
```

Comment:
So this is going sound weird, but GETs dont need the Cookie or Header with the way the server is setup, anything that modifies the data you do. And it defaults to GET, so you could trim it down it a bit
`axios({ url: ... })`